### PR TITLE
Feature add apds 9306 support

### PR DESCRIPTION
**Pull request by Kampi**

### CODE REVIEW

File: app/Kconfig [hunk 1]
```diff
@@ -3,6 +3,48 @@ menu "Zephyr Kernel"
 endmenu
 
 menu "ZSWatch"
+    menu "Applications"
+        config APPLICATIONS_USE_2048
+            bool
+        prompt "Activate the application '2048'"
+        default ny
```
Comment (jakkra) [inline app/Kconfig:10]: ny? Typo?

File: app/drivers/sensor/apds9306/apds9306.c [hunk 2]
```diff
@@ -0,0 +1,438 @@
...
+
+/** @brief          Enable the ALS measurement.
+ *  @param p_Dev    Pointer to sensor device
+ *  @return         0 when successful
+*/
+static int apds9306_enable(const struct device* p_Dev)
+{
+	const struct apds9306_config* Config = p_Dev->config;
+
+	if(i2c_reg_update_byte_dt(&Config->i2c, APDS9306_REGISTER_MAIN_CTRL, ADPS9306_BIT_ALS_EN, ADPS9306_BIT_ALS_EN))
```
Comment (jakkra) [inline app/drivers/sensor/apds9306/apds9306.c:78]: Please do in vscode "convert indentation to spaces" 
one tab == 4 spaces

File: app/Kconfig [hunk 3]
```diff
@@ -3,6 +3,48 @@ menu "Zephyr Kernel"
...
+
+        config APPLICATIONS_USE_QR_CODE
+            bool
+        prompt "Activate the application 'QR-Code'"
+        default y
+
+        config APPLICATIONS_USE_X_RAY
+            bool
+        prompt "Activate the application 'X-Ray'"
+        default y
```
Comment (jakkra) [inline app/Kconfig:40]: Does it link correctly with all on as default?

**Review by jakkra — CHANGES_REQUESTED** [review]
CI not passing. Needs some fixup for that.

File: app/Kconfig (hunk 3 above)
Comment (Kampi) [reply app/Kconfig:40]: You are right. I enabled all per default before pushing the changes and didn´t think about it.

**Pull request merged**